test/ui-test/lightspeedUiTest.ts

```diff
@@ -323,4 +328,26 @@ export function lightspeedUIAssetsTest(): void {
       }
     });
   });
+
+  describe("Verify playbook generation page is not opened when Lightspeed is not enabled", function () {
```

Comment:
I have not added tests for the case Lightspeed is enabled but the user has not yet logged in because I have not yet succeeded to write UI test cases that involve login process (I could not figure out the redirection that is needed for OAuth2 Authorization flow)...